FirebaseAuth/Tests/Unit/SwiftAPI.swift

```diff
@@ -702,4 +702,27 @@ class AuthAPI_hOnlyTests: XCTestCase {
     if let _: Date = metadata.lastSignInDate,
        let _: Date = metadata.creationDate {}
   }
+
+  func regression13429(id: AuthProviderID) -> Int {
+    switch id {
```

Comment:
This is great, but I'm actually confused how this is even allowed. Where are these cases coming from since it's not an enum?